editor/src/components/editor/actions/actions.spec.ts

```diff
@@ -1423,7 +1423,7 @@ describe('INSERT_WITH_DEFAULTS', () => {
                 />
                 <img
                   style={{ width: 100, height: 100 }}
-                  src='/editor/icons/favicons/favicon-128.png?hash=nocommit\\"'
+                  src='/editor/icons/favicons/favicon-128.png?hash=nocommit'
```

Comment:
Huh?

Reply:
There appeared to be an errant `"` on the end of that.